### ISSUE TITLE
Generator losses do not seem to decrease.

### ISSUE THREAD

**Issue by wkhademi** [wkhademi/CycleGAN#2]

**Comment by wkhademi** [comment]
Fixed error in cycle_consistency_loss() in CycleGAN.py

Still unsure if everything with generator loss is fixed as the loss decreases very slowly.

**Comment by wkhademi** [comment]
Realized the original torch code combines the two generator losses and minimizes that rather than minimizing the separate losses.

In CycleGAN.py in get_losses() replaced:
```
   # generator losses  
   G_loss = self.G_loss(self.D_Y, fakeY) + cc_loss
   F_loss = self.G_loss(self.D_X, fakeX) + cc_loss
```

with:
```
   # generator losses
   Gen_loss = self.G_loss(self.D_Y, fakeY) + self.G_loss(self.D_X, fakeX) + cc_loss
```

**Comment by wkhademi** [comment]
After training for some time generator loss is now decreasing as it should.
